? autom4te.cache
? blah.cpp
? irix.patch
? so_locations
? version-script
? wxgtk-2.5-config
? contrib/config.log
? contrib/config.status
? contrib/samples/Makefile
? contrib/samples/applet/Makefile
? contrib/samples/deprecated/Makefile
? contrib/samples/deprecated/proplist/Makefile
? contrib/samples/deprecated/resource/Makefile
? contrib/samples/deprecated/treelay/Makefile
? contrib/samples/fl/Makefile
? contrib/samples/fl/fl_demo1/Makefile
? contrib/samples/fl/fl_demo2/Makefile
? contrib/samples/fl/fl_sample1/Makefile
? contrib/samples/fl/fl_sample2/Makefile
? contrib/samples/fl/fl_sample3/Makefile
? contrib/samples/gizmos/Makefile
? contrib/samples/gizmos/dynsash/Makefile
? contrib/samples/gizmos/dynsash_switch/Makefile
? contrib/samples/gizmos/editlbox/Makefile
? contrib/samples/gizmos/led/Makefile
? contrib/samples/gizmos/multicell/Makefile
? contrib/samples/gizmos/splittree/Makefile
? contrib/samples/mmedia/Makefile
? contrib/samples/ogl/Makefile
? contrib/samples/ogl/ogledit/Makefile
? contrib/samples/ogl/studio/Makefile
? contrib/samples/plot/Makefile
? contrib/samples/stc/Makefile
? contrib/samples/svg/Makefile
? contrib/samples/xrc/Makefile
? contrib/src/Makefile
? contrib/src/applet/Makefile
? contrib/src/deprecated/Makefile
? contrib/src/fl/Makefile
? contrib/src/gizmos/Makefile
? contrib/src/mmedia/Makefile
? contrib/src/net/Makefile
? contrib/src/ogl/Makefile
? contrib/src/plot/Makefile
? contrib/src/stc/Makefile
? contrib/src/svg/Makefile
? contrib/src/xrc/Makefile
? contrib/utils/Makefile
? contrib/utils/convertrc/Makefile
? contrib/utils/wxrc/Makefile
? contrib/utils/wxrcedit/Makefile
? demos/Makefile
? demos/config.log
? demos/config.status
? demos/bombs/Makefile
? demos/dbbrowse/Makefile
? demos/forty/Makefile
? demos/fractal/Makefile
? demos/life/Makefile
? demos/poem/Makefile
? lib/libwx_gtk-2.5.so.0.0.0
? lib/wx
? samples/Makefile
? samples/config.log
? samples/config.status
? samples/nativedlg
? samples/artprov/Makefile
? samples/calendar/Makefile
? samples/caret/Makefile
? samples/checklst/Makefile
? samples/config/Makefile
? samples/console/Makefile
? samples/controls/Makefile
? samples/db/Makefile
? samples/dialogs/Makefile
? samples/dialup/Makefile
? samples/dnd/Makefile
? samples/docview/Makefile
? samples/docvwmdi/Makefile
? samples/dragimag/Makefile
? samples/drawing/Makefile
? samples/dynamic/Makefile
? samples/erase/Makefile
? samples/event/Makefile
? samples/exec/Makefile
? samples/font/Makefile
? samples/grid/Makefile
? samples/help/Makefile
? samples/html/Makefile
? samples/html/about/Makefile
? samples/html/help/Makefile
? samples/html/helpview/Makefile
? samples/html/printing/Makefile
? samples/html/test/Makefile
? samples/html/virtual/Makefile
? samples/html/widget/Makefile
? samples/html/zip/Makefile
? samples/image/Makefile
? samples/internat/Makefile
? samples/ipc/Makefile
? samples/joytest/Makefile
? samples/keyboard/Makefile
? samples/layout/Makefile
? samples/listctrl/Makefile
? samples/mdi/Makefile
? samples/memcheck/Makefile
? samples/menu/Makefile
? samples/minifram/Makefile
? samples/minimal/Makefile
? samples/mobile/Makefile
? samples/mobile/styles/Makefile
? samples/mobile/wxedit/Makefile
? samples/multimon/Makefile
? samples/notebook/Makefile
? samples/oleauto/Makefile
? samples/opengl/Makefile
? samples/opengl/cube/Makefile
? samples/opengl/isosurf/Makefile
? samples/opengl/penguin/Makefile
? samples/ownerdrw/Makefile
? samples/png/Makefile
? samples/printing/Makefile
? samples/propsize/Makefile
? samples/regtest/Makefile
? samples/rotate/Makefile
? samples/sashtest/Makefile
? samples/scroll/Makefile
? samples/scrollsub/Makefile
? samples/shaped/Makefile
? samples/sockets/Makefile
? samples/splitter/Makefile
? samples/statbar/Makefile
? samples/taskbar/Makefile
? samples/text/Makefile
? samples/thread/Makefile
? samples/toolbar/Makefile
? samples/treectrl/Makefile
? samples/typetest/Makefile
? samples/validate/Makefile
? samples/widgets/Makefile
? samples/wizard/Makefile
? utils/Makefile
? utils/config.log
? utils/config.status
? utils/HelpGen/Makefile
? utils/HelpGen/src/Makefile
? utils/dialoged/Makefile
? utils/emulator/Makefile
? utils/emulator/src/Makefile
? utils/helpview/Makefile
? utils/helpview/src/Makefile
? utils/hhp2cached/Makefile
? utils/makegen/Makefile
? utils/tex2rtf/Makefile
? utils/tex2rtf/src/Makefile
Index: include/wx/hashmap.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/include/wx/hashmap.h,v
retrieving revision 1.18
diff -u -r1.18 hashmap.h
--- include/wx/hashmap.h	2003/04/02 11:10:29	1.18
+++ include/wx/hashmap.h	2003/04/14 19:54:23
@@ -16,6 +16,7 @@
 #pragma interface "hashmap.h"
 #endif
 
+#include <stddef.h>  // for ptrdiff_t
 #include "wx/string.h"
 
 // private

 	  	 
